dist/variables/ds4/spinner-variables.less

```diff
@@ -1 +1,2 @@
 @spinner-image-url: "https://ir.ebaystatic.com/cr/v/c1/skin/svg/spinner/v1/ds4/spinner.svg";
```

Comment:
Should I delete these from the dist, or is there a file that tells the dist not to generate these files?

Reply:
I'm pretty sure we should still be keeping files in dist under version control. @agliga can confirm.